### ISSUE TITLE
ERB_MODE Duplicate tables

### ISSUE THREAD

**Issue by leosmeira** [leikind/wice_grid#54]
When i use :erb_mode => true, this generate 2 tables.

I use: RAILS 3.0.9 + inherited_resources + Jquery
## CODE:

<%# Wice_Grid %>
<%= include_wice_grid_assets %>

<% grid(@tabela, :show_filters => :always, :erb_mode => true) do |g| %>

```
<% g.column :column_name => 'Título', :attribute_name => 'titulo' do |tabela| %>
  <%= tabela.titulo %>
<% end  %>

<% g.column :column_name => 'Agência', :attribute_name => 'agencia' do |tabela| %>
  <%= tabela.agencia %>
<% end  %>

<% g.column :column_name => 'Conta', :attribute_name => 'conta' do |tabela| %>
  <%= tabela.conta %>
<% end  %>

<% g.column do |tabela| %>      
    <%= link_to image_tag('/stylesheets/admin/assets/icons/actions_small/display.png'), tabela, :class => 'tooltip table_icon', :title => 'Visualizar' %>
    <%= link_to image_tag('/stylesheets/admin/assets/icons/actions_small/edit.png'), edit_resource_path(tabela), :class => 'tooltip table_icon', :title => 'Editar' %>
    <%= link_to image_tag('/stylesheets/admin/assets/icons/actions_small/delete.png'), tabela, :confirm => 'Deseja excluir este item ?', :method => :delete, :class => 'tooltip table_icon', :title => 'Excluir' %>
<% end %>
```

<% end -%>

**Comment by leosmeira** [comment]
Where can i find this error in the code?

Thank's!

**Comment by leosmeira** [comment]
I am to deliver a project this weekend.
If possible, give attention to this bug.

Thank you!

**Comment by leikind** [comment]
If this is still an issue, it is fixed in version 3.0.4

In any case, a solution could always be switching to the non-ERB, helper-like style, which is a better style anyway
